fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -415,6 +417,134 @@ public void scenario9() throws IOException {
         assertEquals(HttpStatus.SC_FORBIDDEN, getStatus(requestGet3));
     }
 
+    @Test
```

Comment:
It would be helpful to either have a description of what this test does or a link to resource describing the scenario.